charts/seed-operatingsystemconfig/original/templates/osc.yaml

```diff
@@ -38,14 +38,15 @@ spec:
 {{ include "gardener-user" . | indent 2 }}
 {{- if .Values.worker.cri }}
 {{- if eq .Values.worker.cri.name .Values.osc.cri.containerdName }}
-{{ include "containerd-service" . | indent 2 }}
+{{ include "containerd-config-generator" . | indent 2 }}
 {{- end }}
 {{- end }}
   files:
 {{ include "root-certs" . | indent 2 }}
 {{- if .Values.worker.cri }}
 {{- if eq .Values.worker.cri.name .Values.osc.cri.containerdName }}
 {{ include "init-containerd-config-script" . | indent 2 }}
+{{ include "containerd-require-config-generator" . | indent 2 }}
```

Comment:
```suggestion
{{- if .Values.worker.cri }}
{{- if eq .Values.worker.cri.name .Values.osc.cri.containerdName }}
{{ include "containerd-require-config-generator" . | indent 2 }}
{{- end }}
{{- end }}
```

Reply:
Why would that be required? We already have that check in place?